src/control/client/connect.go

```diff
@@ -53,7 +53,7 @@ func (cr ClientResult) String() string {
 	if cr.Err != nil {
 		return fmt.Sprintf("error: " + cr.Err.Error())
 	}
-	return fmt.Sprintf("%+v", cr.Value)
+	return fmt.Sprintf("%#v", cr.Value)
```

Comment:
Is this string only intended to be used for debugging?

Reply:
placeholder for more useful output of generic objects before I get round to finding a proper solution